### PR TITLE
Upgrade neutrino

### PR DESCRIPTION
**Pull request by armenzg**
It seems that we were loosing linting coverage with the setup of rc0.
There's a bunch of automatic fixes and some rules exceptions.
I would like to add the exceptions and then see if a contributor could remove them.
A lot of these linting issues can be seen within Visual Studio if you would like to use that IDE.

### CODE REVIEW

File: src/Home.jsx [hunk 1]
```diff
@@ -39,22 +39,33 @@ const Home = ({ classes }) => (
...
     title="Firefox health"
-    subtitle="Tracking metrics for Firefox products">
+    subtitle="Tracking metrics for Firefox products"
+  >
     <div className={classes.links}>
       <Link to="/android">
-        <AndroidIcon /> Android
+        <AndroidIcon />
+        {' '}
+Android
```
Comment (klahnakoski) [inline src/Home.jsx]: What rule does this!  It is ugly!

File: .neutrinorc.js [hunk 2]
```diff
@@ -1,39 +1,55 @@
...
+          'no-shadow': 0,
+          'no-return-assign': 0,
+          'react/default-props-match-prop-types': 0,
+          'react/destructuring-assignment': 0,
+          'react/forbid-prop-types': 0,
+          'react/no-unused-prop-types': 0,
+          'react/prefer-stateless-function': 0,
+          'react/prop-types': 0,
+          'react/require-default-props': 0,
+          'react/sort-comp': 0,
```
Comment (klahnakoski) [inline .neutrinorc.js]: thank you for figuring out neutrino's extra-complicated "configuration" file.

File: src/android/TP6mAggregate.jsx [hunk 3]
```diff
@@ -112,58 +111,57 @@ async function pullAggregate({
...
-          .limit(8) // 7+1 for the null entry
-          .exists()
-          .isEmpty(),
-    }
+      value: ({ daily }) => !selectFrom(daily)
+        .reverse()
+        .limit(8) // 7+1 for the null entry
+        .exists()
+        .isEmpty(),
+    },
```
Comment (klahnakoski) [inline src/android/TP6mAggregate.jsx]: Does the new linting rules come with automatic removal of comments?

File: src/utils/PerfherderGraphContainer.jsx [hunk 4]
```diff
@@ -176,11 +200,9 @@ const perfherderFormatter = (series, timeDomain) => {
...
       // choose meta from the source with most recent data
       meta: selectFrom(sources)
-        .sort(s =>
-          selectFrom(s.data)
-            .select('push_timestamp')
-            .max()
-        )
+        .sort(s => selectFrom(s.data)
+          .select('push_timestamp')
+          .max())
```
Comment (klahnakoski) [inline src/utils/PerfherderGraphContainer.jsx]: ensuring the opening parenthesis and closing parenthesis do not line up is a mistake

**Comment by armenzg** [comment]
> Thank you for this. These changes, and the others you submitted today, are provided in parallel: Are these going to merge well? Does it all work when applied together?

I would merge one and do manual labour if needed.

File: src/Home.jsx (hunk 1 above)
Comment (armenzg) [reply src/Home.jsx]: It is! This was an automatic fix.
Want me to fix it before merging? Maybe a `span` is needed? I assume that white space is intended.

File: src/android/TP6mAggregate.jsx (hunk 3 above)
Comment (armenzg) [reply src/android/TP6mAggregate.jsx]: I don't know.

File: src/utils/PerfherderGraphContainer.jsx (hunk 4 above)
Comment (armenzg) [reply src/utils/PerfherderGraphContainer.jsx]: We can fix that.

**Comment by armenzg** [comment]
You made few comments but approved the PR.
Please let me know if you need anything address before or merging or good enough to go.

**Comment by klahnakoski** [comment]
@armenzg this PR appears to break `yarn lint --fix`  what is the new command to do the same?

File: src/android/TP6mAggregate.jsx (hunk 3 above)
Comment (klahnakoski) [reply src/android/TP6mAggregate.jsx]: I see that the "conversation" view does not show the comment removed:

image

did you remove it, or did the linter?

File: src/utils/PerfherderGraphContainer.jsx (hunk 4 above)
Comment (klahnakoski) [reply src/utils/PerfherderGraphContainer.jsx]: We can?!

File: src/android/TP6mAggregate.jsx (hunk 3 above)
Comment (armenzg) [reply src/android/TP6mAggregate.jsx]: The linter.

File: src/utils/PerfherderGraphContainer.jsx (hunk 4 above)
Comment (armenzg) [reply src/utils/PerfherderGraphContainer.jsx]: I haven't tried. Maybe.

**Comment by klahnakoski** [comment]
merged with https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/510

**Pull request merged**